Build instructions have no mention of Docker?

Have not managed to get ElectrumWalletSpec to work. As not sure what the Docker config is and is not clear from Travis what it is doing.

I have docker installed, but assume I need to set some env variable or something to get the test code to find it? Getting:

```
00:17:17.593 INFO  ElectrumWalletSpec  - starting bitcoind
Aug 09, 2018 12:17:18 AM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: Permission denied
Aug 09, 2018 12:17:18 AM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}->unix://localhost:80
Aug 09, 2018 12:17:18 AM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: Permission denied
Aug 09, 2018 12:17:18 AM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}->unix://localhost:80
Aug 09, 2018 12:17:18 AM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: Permission denied
Aug 09, 2018 12:17:18 AM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}->unix://localhost:80
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.796 s <<< FAILURE! - in fr.acinq.eclair.blockchain.electrum.ElectrumWalletSpec
```

--- a/BUILD.md
+++ b/BUILD.md
@@ -2,15 +2,20 @@
 
 ## Requirements
 - [Java Development Kit](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) 1.8u161 or newer
-- [Maven](https://maven.apache.org/download.cgi) 3.3.x
+- [Maven](https://maven.apache.org/download.cgi) 3.3.x or newer
 - [Inno Setup](http://www.jrsoftware.org/isdl.php) 5.5.9 (optional, if you want to generate the windows installer)
+- [Docker](https://www.docker.com/) 18.03 or newer (optional) if you want to run all tests
 
 ## Build
 To build the project, simply run:
 ```shell
 $ mvn install
 ```
-To skip the tests, run:
+If you don't have Docker you can skip tests that depend on it:
+```shell
+$ mvn install -DexcludedGroups=fr.acinq.eclair.blockchain.electrum.DockerTest
+```
+To skip all tests, run:
 ```shell
 $ mvn install -DskipTests
 ```

--- a/BUILD.md
+++ b/BUILD.md
@@ -11,6 +11,9 @@ To build the project, simply run:
 ```shell
 $ mvn install
 ```
+
+#### Other build options
+
 If you don't have Docker you can skip tests that depend on it:
 ```shell
 $ mvn install -DexcludedGroups=fr.acinq.eclair.blockchain.electrum.DockerTest

